8.1/jhbuildrc

```diff
@@ -125,5 +125,5 @@ notrayicon = True
 # whether not to emit notifications through the notification daemon
 nonotify = True 
 
-
-
+# jhbuild uses a CPU-meltingly excessive default parallelism of 2x CPU cores
```

Comment:
This comment could be clearer, sorry. "2x CPU cores" means twice the number of CPU cores you have, e.g. a 4x core machine will invoke 8x parallel builds. The use of Docker adds to this CPU hit and my computer (quite rightly) shut itself down when the CPU temperature reached 90C :(